src/pretix/presale/context.py

```diff
@@ -80,10 +81,22 @@ def _default_context(request):
 
     if hasattr(request, 'event') and request.event:
```

Comment:
This might be overkill, but would it make sense to check, if any customization on event-level was done? I guess it is safe to assume that at least on organizer-level there are customizations, but on event-level? That way we would not need to compile (and re-compile) for every event of an organizer, but most of the time just on organizer-level and just include the organizer-theme.css on events as well. Problem is, how do we check settings are on organizer-level and not event-level when event-settings inherit from organizer automagically … Maybe not worth the effort?